recipe/meta.yaml

```diff
@@ -26,7 +26,6 @@ outputs:
     requirements:
       build:
         - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
```

Comment:
I think you still need this for configure to run and to support the `mpicxx` compiler wrapper.